.ci/pipeline/coverity.yml

```diff
@@ -0,0 +1,27 @@
+variables:
+  COVERITY_TOOL_HOME: $(Agent.BuildDirectory)/cov-analysis
+  DESCRIPTION: Nightly
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+- script: |
+    cd $(Agent.BuildDirectory)
+    wget https://scan.coverity.com/download/linux64 --post-data "token=$(COVERITY_AUTHKEY)&project=daal4py" -O coverity_tool.tgz
```

Comment:
Coverity recommendation to get this cashed instead of downloading each time from their server. 
For example you can use this as an example - https://github.com/oneapi-src/oneDAL/blob/master/.ci/pipeline/ci.yml#L83

Reply:
We will enable caching at the nearest time, thanks for information and example. For now it is more preferable to enable continious scan.